source/extensions/transport_sockets/http_11_proxy/connect.cc

```diff
@@ -15,14 +15,15 @@ namespace Extensions {
 namespace TransportSockets {
 namespace Http11Connect {
 
-bool UpstreamHttp11ConnectSocket::isValidConnectResponse(Buffer::Instance& buffer) {
+bool UpstreamHttp11ConnectSocket::isValidConnectResponse(absl::string_view data, bool& error, unsigned long& len) {
```

Comment:
It looks like Parse::execute() returns a size_t so we should probably use that here. (Though that method should probably be modified to return uint64_t as per current guidance, but that's way outside the scope of this PR.

Reply:
nit: Consider renaming `len` to `bytes_parsed` or some such for enhanced clarity.